content/en/logs/log_configuration/processors.md

```diff
@@ -565,22 +563,25 @@ With the following log, use the template `Request %{http.method} %{http.url} was
 }
 ```
 
+Returns the following:
+
 ```text
 Request GET https://app.datadoghq.com/users was answered with response 200
 ```
 
-**Note**: `http` is an object and cannot be used in a block (`%{http}` fails), whereas `%{http.method}`, `%{http.status_code}`, or `%{http.url}` returns the corresponding value. Blocks can be used on arrays of values or on a specific attribute within an array. For example, adding the block `%{array_ids}` returns:
+**Note**: `http` is an object and cannot be used in a block (`%{http}` fails), whereas `%{http.method}`, `%{http.status_code}`, or `%{http.url}` returns the corresponding value. Blocks can be used on arrays of values or on a specific attribute within an array. 
 
-```text
-123,456,789
-```
+* For example, adding the block `%{array_ids}` returns:
 
-Whereas `%{array_users}` does not return anything because it is a list of objects.
-However, `%{array_users.first_name}` returns a list of `first_name` contained in the array:
+   ```text
+   123,456,789
+   ```
 
-```text
-John,Jack
-```
+* Whereas `%{array_users}` does not return anything because it is a list of objects. However, `%{array_users.first_name}` returns a list of `first_name` contained in the array:
```

Comment:
```suggestion
* `%{array_users}` does not return anything because it is a list of objects. However, `%{array_users.first_name}` returns a list of `first_name`s contained in the array:
```